cartography/intel/aws/redshift.py

```diff
@@ -77,9 +78,10 @@ def load_redshift_cluster_data(neo4j_session, clusters, region, current_aws_acco
             aws_update_tag=aws_update_tag,
         )
         _attach_ec2_security_groups(neo4j_session, cluster, aws_update_tag)
```

Comment:
do these need to be run in order? If not, using gevent would make these significantly faster.  If latency isn't a big deal then no biggie but fast is nice

Reply:
For these `_attach*()` functions, no. I'll look into gevent, thanks!

Reply:
Actually, I'll experiment with gevent later on a bigger bottleneck. In this particular case the `_attach()` functions don't take much time at all.